test/dummy/spec/models/upload_spec.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+require "spec_helper"
+
+describe Goldencobra::Upload do
+  before(:all) do
+    @zip = Goldencobra::Upload.create(image: File.open(File.join(Rails.root, "spec/fixtures/files/test_archive.zip")))
+    @zip_severals = Goldencobra::Upload.create(image: File.open(File.join(Rails.root, "spec/fixtures/files/test_archive_severals.zip")))
+    @zip_subfolders = Goldencobra::Upload.create(image: File.open(File.join(Rails.root, "spec/fixtures/files/test_archive_subfolders.zip")))
+  end
+
+  it "can upload and unzip a zip file" do
+     @zip.unzip_files
```

Comment:
This test does not `expect` anything and validates nothing, does it?
Do you know this syntax https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher#expect-no-error-at-all ?